poster/styles/styles.scss

```
@@ -204,53 +353,67 @@ ul{
 }
 
 .link__instagram{
+  display: block;
   font-family: $font-condensed;
   font-size: 0.6rem;
   font-weight: 500;
   letter-spacing: 0.1rem;
   margin: 2rem;
   text-transform: lowercase;
+  
+  @media only screen and (min-width: 900px){
+    font-size: 0.8rem;
+  }
 }
 
+// small circles with info
+
 .info-circles{
   grid-column: 3/4;
   grid-row: 3/6;
-  font-size: 7px;
-  line-height: 9px;
+  line-height: 0.62rem;
+
+  @media only screen and (max-width: 700px){
+    grid-column: 2/3;
+  }
 }
 
 .info-circles__date{
-  @include size(45px,45px);
-  border-radius: 25px;
+  @include size(3rem,3rem);
+  border-radius: 3rem;
   border: 0.1rem solid $color-black;
-  margin: 0 0 10px 0;
-  padding-top: 8px;
+  font-size: 0.05rem;
+  margin: 0 0 0.9rem 0;
```

I recommend not using the shorthand property when you only want to set one of the values, so this would be better:
```suggestion
  margin-bottom: 0.9rem;
```

Same idea is true for the margin and padding rules on lines 387, 397, 407, 408, 421